src/node.cc

```diff
@@ -4202,8 +4208,14 @@ int Start(int argc, char** argv) {
   V8::SetEntropySource(crypto::EntropySource);
 #endif
 
-  const int thread_pool_size = 4;
-  default_platform = v8::platform::CreateDefaultPlatform(thread_pool_size);
+  int pool_size = v8_default_thread_pool_size;
+
+  if (v8_thread_pool_size != nullptr) {
+    // if the parameter is given, try to convert the string to an int.
+    pool_size = atoi(v8_thread_pool_size);
+  }
+
+  default_platform = v8::platform::CreateDefaultPlatform(pool_size);
```

Comment:
Should we maybe emit warnings here then if the size is too big/small?

i.e. that the threadpool max is 4 if it is larger than that, and also something else if the user specified `<1`